qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1520A.py

```diff
@@ -89,6 +92,22 @@ def _set_voltage_ac(self, value: float) -> None:
 
         self.write(msg.message)
 
+    def _get_dcv(self):
+        if self.is_enabled():
+            msg = MessageBuilder().lrn_query(self.channels[0])
+            response = self.ask(msg.message)
+            d = parse_dcv_measurement_response(response)
+        return d or None
```

Comment:
does it make sense to allow returning ``None``? i'd say if `is_enabled` is `False`, then just raise an error. This will also simplify the get methods for the individual parameters